cri/v1alpha2/container_pause.go

```diff
@@ -0,0 +1,31 @@
+package v1alpha2
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	runtime "github.com/alibaba/pouch/cri/apis/v1alpha2"
+	"github.com/alibaba/pouch/cri/metrics"
+	util_metrics "github.com/alibaba/pouch/pkg/utils/metrics"
+)
+
+// PauseContainer pauses the container.
+func (c *CriManager) PauseContainer(ctx context.Context, r *runtime.PauseContainerRequest) (*runtime.PauseContainerResponse, error) {
+	label := util_metrics.ActionPauseLabel
+	metrics.ContainerActionsCounter.WithLabelValues(label).Inc()
+	defer func(start time.Time) {
+		metrics.ContainerActionsTimer.WithLabelValues(label).Observe(time.Since(start).Seconds())
+	}(time.Now())
+
+	containerID := r.GetContainerId()
+
+	err := c.ContainerMgr.Pause(ctx, containerID)
```

Comment:
nit, you can use `if err := c.ContainerMgr.Pause(ctx, containerID); err != nil` here.